docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/connected_cluster/_index.md

```diff
@@ -0,0 +1,132 @@
+
+  > **NOTE: This scenario assumes you already deployed a Kubernetes cluster and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion**
+
+- **[Azure Kubernetes Service](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aks/)**
+- **[AKS on Azure Stack HCI](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/)**
+- **[Kubernetes Cluster API](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/cluster_api/)**
+- **[Azure Red Hat OpenShift](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aro/)**
+- **[Amazon Elastic Kubernetes Service](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/eks/)**
+- **[Google Kubernetes Engine](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/gke/)**
+- **[Alibaba Cloud Container Service for Kubernetes](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/alibaba/)**
+- **[Rancher K3s](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/rancher_k3s/)**
+- **[Kind](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/kind/)**
+- **[MicroK8s](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/microk8s/)**
+- **[Platform9 Managed Kubernetes](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/pf9/)**
+
+## Prerequisites
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- Ensure you are logged in to the Azure CLI or PowerShell
+
+  ```shell
+  az login
+  ```
+
+  ```powershell
+  Connect-AzAccount
+  ```
+
+- [Install or update Azure CLI to version 2.42.0 and above](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+- [Install or update to the latest version of the _connectedk8s_ CLI extension](https://learn.microsoft.com/azure/azure-arc/kubernetes/cluster-connect?tabs=azure-cli).
+
+  ```shell
+  az extension add --name connectedk8s
+  ```
+
+- As mentioned, this scenario starts at the point where you already have an Azure Arc connected cluster.
+
+    ![Screenshot showing existing Azure Arc connected cluster](./01.png)
+
+## Azure Arc Kubernetes Connected Cluster Configuration
+
+The following steps walk through using the Cluster Connect functionality using Azure Active Directory (AAD); however, it can also be done using a service account token.  The benefit of using AAD authentication is that your current, logged-in user from the Azure CLI will be used and you won't have to obtain a service account token from your cluster.
+
+- First, get values for the cluster name and resource group name in your terminal.
+
+  ```shell
+  CLUSTER_NAME=<cluster-name>
+  RESOURCE_GROUP=<resource-group-name>
+  ```
+
+  ```powershell
+  $CLUSTER_NAME = <cluster name>
+  $RESOURCE_GROUP = <resource_group>
+  ```
+
```

Comment:
```suggestion

PowerShell:
  ```powershell
  $CLUSTER_NAME = <cluster name>
  $RESOURCE_GROUP = <resource_group>
  ```

```

Reply:
Wherever applicable, add Bash/PowerShell command indicator before the command. Otherwise, one can be confused by the two code blocks